index.html

```diff
@@ -284,22 +284,22 @@ <h4>A lot of Useful Tools</h4>
 
 			<!-- Start callto-action Area -->
 			<section class="callto-action-area relative section-gap">
-				<div class="overlay overlay-bg"></div>	
+				<div class="overlay overlay-bg"></div>
 				<div class="container">
 					<div class="row d-flex justify-content-center">
 						<div class="menu-content col-lg-9">
 							<div class="title text-center">
 								<h1 class="mb-10 text-white">Not Impressed Yet?</h1>
-								<p class="text-white">This OS may not be for everybody. It only targets a specific user base. People who are new to linux, people who like to customize their desktop, people who cares about how things looks on their system, distohoppers.
-								If you're a geek, an expert, an experienced linux user, or the one who prefer to configure everything from scratch, this may not be for you.<br> But anyway if you're interested...</p>
+								<p class="text-white">This OS may not be for everybody. It's for people who are new to linux, like to customize their desktop or people who care more about their system's aesthetics.
```

Comment:
Linux (capitalised)

Reply:
…or *those* who care…?

Reply:
Thanks for pointing out, you got a keen eye!  Will change both.